test/CMakeLists.txt

```diff
@@ -88,15 +88,27 @@ if (APPLE)
     set(NGRAPH_TF_CXX11_ABI 0)
 endif()
 
-target_link_libraries(
-    gtest_ngtf
-    ngraph_bridge
-    ngraph_lib
-    libgtest
-    pthread
-    tensorflow_framework_lib
-    tensorflow_cc_lib
-)
+if (NGRAPH_BRIDGE_STATIC_LIB_ENABLE)
+    target_link_libraries(
+        gtest_ngtf
+        ngraph_bridge_static
+        ngraph_lib
+        libgtest
+        pthread
+        tensorflow_framework_lib
+        tensorflow_cc_lib
+    )
+else()
+    target_link_libraries(
+        gtest_ngtf
+        ngraph_bridge
+        ngraph_lib
+        libgtest
+        pthread
+        tensorflow_framework_lib
+        tensorflow_cc_lib
+    )
+endif()
```

Comment:
suggestion:

```
target_link_libraries(
        gtest_ngtf
        BRIDGE_LIB_TO_LINK_FOR_TEST
        ngraph_lib
        libgtest
        pthread
        tensorflow_framework_lib
        tensorflow_cc_lib
    )
```

and in src's cmake near the `if (NGRAPH_BRIDGE_STATIC_LIB_ENABLE)` we add `set(BRIDGE_LIB_TO_LINK_FOR_TEST, LIB_NAME_STATIC)` (or LIB_NAME as necessary)

or something similar. I do not know enough cmake of the OTTOMH